src/browser/modules/Sidebar/project-files.utils.ts

```diff
@@ -119,6 +122,55 @@ export const createFilePath = (paths: string[]): string => {
   return paths.join(/Win32/.test(navigator.platform) ? '\\' : '/')
 }
 
+export const checkFileNameInput = (fileName: string): string => {
+  if (!fileName.length) {
+    return 'File name cannot be empty'
+  }
+
+  const errors: ProhibitedFilenameErrors = { chars: [], tests: [] }
+
+  PROHIBITED_FILENAME_CHAR_TESTS.forEach(charObj => {
+    if (charObj.test) {
```

Comment:
Not strictly needed but would it not be easier if they were all regex tests?

Reply:
yeah that makes sense actually